actions/mouths-of-watersheds/DESCRIPTION.md

```diff
@@ -0,0 +1,22 @@
+# Find Mouth of Watershed
+
+This script locates the mouth of a watershed from a geojson file by matching
+it up with a netCDF file. The resulting mouth of the watershed along with
+the watershed code are outputted as a CSV file.
```

Comment:
This description is helpful but a little vague. We can make it more specific by saying something like

Locates the mouth of a watershed whose boundary is defined by a polygon(?) in a GeoJSON file by finding an interior point in the watershed and following that point to its drainage point as defined by a [insert proper terminology here] flow matrix provided in a NetCDF file.